transport/src/main/java/io/netty/channel/nio/AbstractNioMessageChannel.java

```diff
@@ -175,6 +171,15 @@ protected boolean continueOnWriteError() {
         return false;
     }
 
+    protected boolean closeOnReadError(Throwable cause) {
+        if (cause instanceof IOException && !(cause instanceof PortUnreachableException)) {
```

Comment:
@normanmaurer what about `RuntimeException`s or `Error`s? Should we close in that case as well?